frontend/src/components/Simulator/TransactionItem.vue

```diff
@@ -60,30 +65,52 @@ watch(
 );
 
 function prettifyTxData(x: any): any {
+  const oldResult = x?.consensus_data?.leader_receipt?.result;
```

Comment:
this ugly code tries to prettify values (calldata and results) for user in json viewer